### ISSUE TITLE
Styling popup window not works

### ISSUE THREAD

**Issue by Suggorath** [thomasloven/hass-browser_mod#335]
My Home Assistant version:
- Home Assistant Core 2022.6.5
- Home Assistant Supervisor 2022.05.3
- Home Assistant OS 8.1
- HACS 1.25.5
- button-card 3.4.2
- browser-mod 1.5.3

**What I am doing:**
I am trying styling browser-mod popup window `.mdc-dialog__surface`

**What happened instead:**
nothing

```yaml
cards:
  - type: custom:button-card
    entity: sensor.plug_computer_energy_today
    tap_action:
      action: fire-dom-event
      browser_mod:
        command: popup
        title: Styled popup
        card:
          type: markdown
          content: Popup!
        style: |
          .mdc-dialog__surface {
            background: transparent !important;
          }
```
- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by petoblat** [comment]
> I reinstall all modules what I have and problem was solved.

What do you mean?
I reinstall Browser_mod from HACS but the problem continue to exist.

**Comment by Suggorath** [comment]
Try reinstall "**card-mod 3**" too. **https://github.com/thomasloven/lovelace-card-mod**

**Comment by petoblat** [comment]
> Try reinstall "**card-mod 3**" too. **https://github.com/thomasloven/lovelace-card-mod**

None. Doesn't work :(

**Comment by thomasloven** [comment]
Closed because of Browser Mod 2.0 release.
Please see #354.
